03b_model/src/run_model.py

```diff
@@ -557,10 +561,10 @@ def make_predictions(prepped_model_io_data_file, target,
     #unnormalize
     #predictions for train val set
     preds_trainval_c  = preds_trainval.detach().numpy().reshape(preds_trainval.shape[0]*preds_trainval.shape[1],preds_trainval.shape[2])
-    unnorm_trainval = ((preds_trainval_c*means_stds['y_std_trnval'][target].data)+means_stds['y_mean_trnval'][target].data).squeeze()
+    unnorm_trainval = ((preds_trainval_c*means_stds['y_std_trn'][target].data)+means_stds['y_mean_trn'][target].data).squeeze()
```

Comment:
I'm not sure what this switch was - is this something that should be set in the config file if it will be toggled?

Reply:
This is a small change to how the data are normalized and unnormalized before and after training, respectively. They were initially normalized/unnormalized with the training and validation data, but now it is just with the training data. It shouldn't have a big effect and I don't think we'll need to change/toggle it.